ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/DrawableDB.java

```diff
@@ -391,14 +391,6 @@ private boolean initializeDB() {
             return false;
         }
 
-        // @@@ BC: I don't think this is needed because it is the primary key. 
-        try (Statement stmt = con.createStatement()) {
```

Comment:
does it hurt to have this.  presumably it is a know op if the primary key counts as an index. but yes it seems redundant